gammapy/astro/darkmatter/spectra.py

```diff
@@ -60,11 +90,8 @@ def mDM(self, mDM):
     @property
     def allowed_channels(self):
         """Allowed annihilation channels
-
-        TODO: Introduce a dict with simpler channel names
-        TODO: Make allowed_channels a class property
         """
-        return self.table.colnames[2:]
+        return self.channel_registry.keys()
```

Comment:
Remove `allowed_channes` and instead document `channel_registry`?

Most users don't know what a dict keys object is, and it's not safe to hand it around in case someone updates the dict between making the dict keys object and using it I think. So if you keep this you should do `return list(self.channel_registry.keys())` instead. But it's just not needed, no?